### DISCUSSION TITLE
How to add boorus ?

### DISCUSSION THREAD

**Discussion by tazeps** [flexbooru/flexbooru#190] [Q&A]
I installed the google play app but i don't know what to do.
I added gelbooru to the account management but no posts show up.

when I go into the tags tab I see the list of tags but when I click on them no posts show up. even if I search for something.

what do I do ? and how do I add booru's properly ? and use the QR code ?

**Comment by Ruvium** [comment]
Here, click the links:
https://flexbooru.github.io/flexbooru/config.html
it was in the telegram channel
